package-lock.json

```diff
@@ -205,6 +205,64 @@
         "purgecss": "^3.1.3"
       }
     },
+    "node_modules/@jridgewell/gen-mapping": {
```

Comment:
why this file changed?

Reply:
When debugging, I ran a general npm build. So now it's updated to latest versions.